site2/docs/schema-evolution-compatibility.md

```diff
@@ -155,7 +155,7 @@ In some data formats, for example, Avro, you can define fields with default valu
 
 :::tip
 
-You can set schema compatibility check strategy at the topic, namespace or broker level. For how to set the strategy, see [here](schema-manage.md/#set-schema-compatibility-check-strategy).
+You can set schema compatibility check strategy at the topic, namespace or broker level. For how to set the strategy, see [here](schema-manage.md#set-schema-compatibility-check-strategy).
```

Comment:
Preview
![image](https://user-images.githubusercontent.com/989425/169820359-5b58ac6e-bced-4de6-9189-975853b8004a.png)

Leading to 
![image](https://user-images.githubusercontent.com/989425/169820411-e3b54875-e585-4f06-ab76-5b3188d3dc73.png)